infra/deploy.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+if [[ -z ${GIT_TAG} ]]; then
+	echo "Missing git tag"
+	exit 1
+fi
```

Comment:
It might also be worth checking if the repo is "clean" as well (has uncommitted changes) - particularly for the k8s files in the `kubectl apply` case.

Reply:
good point, done